src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/books/addBook';
+const REMOVE_BOOK = 'bookstore/books/removeBook';
+
+const initialState = {
+  books: [],
+};
+
+// Add Reducers
+export default function booksReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return { ...state };
+    case REMOVE_BOOK:
+      return {
+        ...state,
+        items: state.items.filter((item) => item.id !== action.id),
+      };
+    default:
+      return state;
+  }
+}
+
+export const addBook = () => ({
+  type: ADD_BOOK,
+});
```

Comment:
- [ ] KIndly make sure your `addBook` action is receiving a new book object as a parameter which will be added in the current state as a new added book.

    - Suggestion:
       `export const addBook = (book) => ({
                type: ADD_BOOK,
                payload: book
         })`